ui/app/src/assets/guestMessages.ts

```diff
@@ -161,5 +161,10 @@ export const guestMessages = defineMessages({
   uploadError: {
     id: 'desktopAssetUpload.uploadError',
     defaultMessage: 'Upload error. File was not uploaded'
+  },
+  outdatedExpBuilderVersion: {
+    id: 'validations.outdatedExpBuilderVersion',
+    defaultMessage:
+      "Your application's Experience Builder package version is out of date. Please notify the development team to perform the update to avoid issues."
```

Comment:
Please update the string to what I indicated in the prior comment.

Reply:
This is the string:

```
Your application's Experience Builder package version is out of date. Please notify the development team to perform the necessary updates to avoid any possible issues.
```